### PR TITLE
Handle wildcard parameter in Topic deletion API

### PR DESCRIPTION
**Pull request by ThomasCAI-mlv**

### CODE REVIEW

File: src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java [hunk 1]
```diff
@@ -203,6 +203,25 @@ public void deleteTopic(Topic topic) throws InterruptedException, ExecutionExcep
             managedClusterProperties.getName());
     }
 
+    /**
+     * Delete a list of topics.
+     *
+     * @param topics The topics to delete
+     */
+    public void deleteTopics(List<Topic> topics) throws InterruptedException, ExecutionException, TimeoutException {
```
Comment (loicgreffier) [inline src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java:211]: Little improvement:
https://github.com/michelin/ns4kafka/blob/ae3e5d9a145a6869c304a56d5dbda0bbb5f4ea37/src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java#L197 can call `deleteTopics(List<Topics> topics)` like

public void deleteTopic(Topic topic) {
  deleteTopics(List.of(topic));
}

**Review by loicgreffier — COMMENTED** [review]
@ThomasCAI-mlv Just one improvement, otherwise LGTM

File: src/main/java/com/michelin/ns4kafka/controller/topic/TopicController.java [hunk 2]
```diff
@@ -156,16 +156,55 @@ public HttpResponse<Topic> apply(String namespace, @Valid @Body Topic topic,
...
+
     /**
      * Delete a topic.
      *
      * @param namespace The namespace
      * @param topic     The topic
-     * @param dryrun    Is dry run mode or not ?
+     * @param dryrun    Is dry run mode or not?
      * @return An HTTP response
+     * @deprecated use bulkDelete(String, String name, boolean) instead.
```
Comment (loicgreffier) [inline src/main/java/com/michelin/ns4kafka/controller/topic/TopicController.java:203]: `{@link #bulkDelete(String, String, boolean)}` better 😉

File: src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java (hunk 1 above)
Comment (ThomasCAI-mlv) [reply src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java:211]: Maybe, it is even better to completely remove the method and replace it by `deleteTopics(List<Topic> topics)`.
What do you think?

**Pull request merged**